README.md

```diff
@@ -18,82 +42,109 @@ There's currently a server (https://github.com/haadcode/orbit-server) that track
 ![Screenshot 2](https://raw.githubusercontent.com/haadcode/orbit/master/screenshots/screenshot3%202016-04-14.png)
 ![Screenshot 3](https://raw.githubusercontent.com/haadcode/orbit/master/screenshots/screenshot6%202016-04-17.png)
 
-## Requirements
-- Node.js v4.x.x
-- npm
+## Install
+
+First, clone this repository locally. Then:
+
+```bash
+npm install
+```
+
+### Requirements
+- [Node.js v4.x.x](http://nodejs.org/)
+- [npm](https://npmjs.com)
```

Comment:
would be nice to include the version of npm required, as npm@2 and npm@3 works differently